lib/Case.ark

```diff
@@ -0,0 +1,119 @@
+{
+    # Made with <3 by Natend (Natendrtfm on github)
+    (let toLowerCase (fun (text) {
+        (mut i 0)
+        (mut element "")
+        (mut output "")
+        (while (< i (len text)) {
+            (set element (@ text i))
+            #Conditions
+            (if (= element "A") {(set element "a")} ())
+            (if (= element "B") {(set element "b")} ())
+            (if (= element "C") {(set element "c")} ())
+            (if (= element "D") {(set element "d")} ())
+            (if (= element "E") {(set element "e")} ())
+            (if (= element "F") {(set element "f")} ())
+            (if (= element "G") {(set element "g")} ())
+            (if (= element "H") {(set element "h")} ())
+            (if (= element "I") {(set element "i")} ())
+            (if (= element "J") {(set element "j")} ())
+            (if (= element "K") {(set element "k")} ())
+            (if (= element "L") {(set element "l")} ())
+            (if (= element "M") {(set element "m")} ())
+            (if (= element "N") {(set element "n")} ())
+            (if (= element "O") {(set element "o")} ())
+            (if (= element "P") {(set element "p")} ())
+            (if (= element "Q") {(set element "q")} ())
+            (if (= element "R") {(set element "r")} ())
+            (if (= element "S") {(set element "s")} ())
+            (if (= element "T") {(set element "t")} ())
+            (if (= element "U") {(set element "u")} ())
+            (if (= element "V") {(set element "v")} ())
+            (if (= element "W") {(set element "w")} ())
+            (if (= element "X") {(set element "x")} ())
+            (if (= element "Y") {(set element "y")} ())
+            (if (= element "Z") {(set element "z")} ())
+            #Special letters (À Á Â Ä Å Ã Æ Ç É È Ê Ë Í Ì Î Ï Ñ Ó Ò Ô Ö Ø Õ OE Ú Ù Û Ü Ý Y)
+            (if (= element "Â") {(set element "â")} ())
+            (if (= element "À") {(set element "à")} ())
+            (if (= element "Á") {(set element "á")} ())
+            (if (= element "Ã") {(set element "ã")} ())
+            (if (= element "Ä") {(set element "ä")} ())
+            (if (= element "Å") {(set element "å")} ())
+            (if (= element "Æ") {(set element "æ")} ())
+            (if (= element "Ç") {(set element "ç")} ())
+            (if (= element "È") {(set element "è")} ())
+            (if (= element "É") {(set element "é")} ())
+            (if (= element "Ê") {(set element "ê")} ())
+            (if (= element "Ë") {(set element "ë")} ())
+            (if (= element "Ì") {(set element "ì")} ())
+            (if (= element "Í") {(set element "í")} ())
+            (if (= element "Î") {(set element "î")} ())
+            (if (= element "Ï") {(set element "ï")} ())
+            (if (= element "Ô") {(set element "ô")} ())
```

Comment:
The `{(set element "...")}` can be reduced to `(set element "...")`

For performance reason it would be better to have something like
```
(if cond
    then
(if cond2
    then
    ...
))))
```